meetings/2023-04-19.md

```diff
@@ -0,0 +1,131 @@
+* PA: +1 to the consistency argument. We must be consistent in the process. We have to mention these CG specifications that are currently normative references. What we should not commit to in the charter is whether those references have to become specifications of their own. The idea is that the separation in documents might not match the current separation of the CG specifications. The charter should give a high-level view of what the WG is expected to address. We should not be too specific in how these references are to be used in the final result.
+* SC: Would you say that those specifications that are listed in the purview of the Solid CG should be listed as a tentative deliverable in the charter, as they currently are in the PR?
+* EP: There might be a benefit in not being over-specific in how we are to deliver the scope. For example it might be beneficial to extract the storage definition, similar to WAC, ACP, ..., where the protocol groups these different specifications. Maybe we just don't have to commit yet at the start on the how, and focus on the scoping right now.
+* EP: Question to PA: Looking at the deliverables section in the [charter](http://htmlpreview.github.io/?https://github.com/solid/solid-wg-charter/blob/12a9e08ff5876e24c0f1b7c0f180d3e1bc5b6efa/charter/index.html) how specific should we be?
+* EP: Currently the deliverables section matches the organization of the specifications in the CG. Isn't this over-specific for the charter?
+* PA: I think it is good to point to existing work that is in scope. Which is the goal of these pointers in the charter. There is a strong dependency between the specification and these references. I don't have a strong opinion on whether this should be in the scope or deliverables section.
+* SC: Just a reminder that they were introduced based on prior observations on other charters / formal objections (e.g. [Devices and Sensors WG](https://www.w3.org/groups/wg/das), Web Application WG). See past discussion: https://github.com/solid/specification/blob/main/meetings/2023-04-12.md#clarify-normative-tentative-new-deliverables. The general idea is that if it is not sufficiently incubated, it should be listed as an in-progress or tentative specification that the WG could consider as a deliverable.
+* EP: We could have a list of more adopted drafts, e.g., WAC, ACP, etc. We cannot expect the final protocol to have the functionality, if it is not in the scope of the WG.
+* SC: Suggestion that PA review the PR and validates whether this is along the lines of the W3C guidelines and is sufficient to wrap up the deliverable.
+* PA: Yes, I will review this PR and try to get feedback from other W3C people about how to address this. I like the idea to list them as other "adopted" drafts, however this might indicate a stronger commitment. 
```

Comment:
```suggestion
* SC: Suggestion that PA review the PR and validate whether this is along the lines of the W3C guidelines and is sufficient to wrap up the deliverable.
* PA: Yes, I will review this PR and try to get feedback from other W3C people about how to address this. I like the idea to list them as other "adopted" drafts; however, this might indicate a stronger commitment. 
```